src/test/java/teammates/test/cases/util/StringHelperTest.java

```diff
@@ -124,6 +125,23 @@ public void testKeyEncryption() {
     }
 
     @Test
+    public void testDecryptingInvalidCiphertextThrowsException() {
+        String invalidHexString = "GHI";
+        // each hex digit is 4 bits
+        String ciphertextLength120 = "AAAAAAAAAABBBBBBBBBBCCCCCCCCCC";
+        String ciphertextLength136 = ciphertextLength120 + "1234";
+        String[] invalidCiphertexts = {invalidHexString, ciphertextLength120, ciphertextLength136};
```

Comment:
@damithc The decrypt function converts a hex string into an array of bytes before they are decrypted, so e.g AF is the byte  10101111. So non-hex strings should fail to decrypt. 

AES requires the length of data to be multiples of 128 bits. So it checks if it fails on inputs of 120 and 136 bits.